### PR TITLE
Bridge config topic node template

### PR DESCRIPTION
**Pull request by gsychev**
Adds ${node} interpolation in topic option of configuration
Just like the clientid already works

Closes: https://github.com/emqx/emqx/issues/6431

> Cherry-picked from #6992 because I forked from the wrong branch

### CODE REVIEW

File: apps/emqx_bridge_mqtt/src/emqx_bridge_mqtt.appup.src [hunk 1]
```diff
@@ -1,24 +1,20 @@
...
-    ]},
-    {"4.3.0", [
-      {load_module, emqx_bridge_worker, brutal_purge, soft_purge, []},
-      {load_module, emqx_bridge_mqtt_actions, brutal_purge, soft_purge, []}
-    ]},
-    {<<".*">>, []}
-  ]
-}.
+%% -*- mode: erlang -*-
+{VSN,
```
Comment (savonarola) [inline apps/emqx_bridge_mqtt/src/emqx_bridge_mqtt.appup.src:2]: Here we should have correct version.

File: apps/emqx_bridge_mqtt/src/emqx_bridge_mqtt.erl [hunk 2]
```diff
@@ -37,6 +37,9 @@
         , handle_disconnected/2
         ]).
 
+%% for testing
```
Comment (savonarola) [inline apps/emqx_bridge_mqtt/src/emqx_bridge_mqtt.erl:40]: Here we better use the common pattern:
```
-ifdef(TEST).
-export([....]).
-endif.
```

Comment (gsychev) [reply apps/emqx_bridge_mqtt/src/emqx_bridge_mqtt.erl:40]: Thanks, I didn't know that

File: apps/emqx_bridge_mqtt/src/emqx_bridge_mqtt.appup.src (hunk 1 above)
Comment (thalesmg) [reply apps/emqx_bridge_mqtt/src/emqx_bridge_mqtt.appup.src:2]: I believe that rebar3 substitutes this `VSN` with the app version from `app.src` (there are other appup's that do the same)

Comment (gsychev) [reply apps/emqx_bridge_mqtt/src/emqx_bridge_mqtt.appup.src:2]: @thalesmg So what should I change and how?
I've read this doc

https://www.erlang.org/doc/man/appup.html

Comment (gsychev) [reply apps/emqx_bridge_mqtt/src/emqx_bridge_mqtt.appup.src:2]: Great, thanks!

**Comment by HJianBo** [comment]
Hi, @gsychev These changes are good for me. 
Here's a small suggestion: We should keep the commit tree as clear as possible. Please squash some related commits.

**Review by HJianBo — APPROVED** [review]

**Review by zmstone — DISMISSED** [review]

**Comment by savonarola** [comment]
Seems there two commits left, let's them squash in a single commit with the title reflecting the feature introduced.

**Review by mononym — APPROVED** [review]

**Comment by zmstone** [comment]
Oops there is a conflict

**Comment by HJianBo** [comment]
Let's merge this PR as soon as possible!! There are some issues to fix:
1. Conflicts should be resolved
2. [This Commit Log](https://github.com/emqx/emqx/pull/7014/commits/0bd080c063dc0f499bce52fad4b6b9f7e51f96fa) seems to be a bit messy, can you delete these lines?
```
feat(emqx_bridge_mqtt): bumped versions the right way

chore(appup): appups for mqtt bridge

feat(mqtt_bridge): test for bridge config

And also bumped one small version

chore(mqtt_bridge): updated CHANGES-4.3.md

fix(mqtt_bridge): conditional export

chore(mqtt_bridge): appup
```

**Review by zmstone — APPROVED** [review]
nit:
* rebase is preferred
* ensure newline at the last line.

**Pull request merged**